clang/runtime/dpct-rt/include/dpct/codepin/codepin.hpp

```diff
@@ -23,14 +23,27 @@
 #include <unordered_set>
 #include <utility>
 #include <vector>
+#include <stdlib.h>
+
+#ifndef CODEPIN_RAND_SEED
+#define CODEPIN_RAND_SEED 0
+#endif
+
+#ifndef CODEPIN_SAMPLING_THRESHHOLD
+#define CODEPIN_SAMPLING_THRESHHOLD 20
```

Comment:
Please add comments on the 3 macros on:  What it is, When to use, how to use;

Reply:
2. how about:  emit build time warnning message to notify the default samping is enabled, and how to change the default sampling?